### PR TITLE
Update step for the section 'Create the Testing Notes'

### PR DESCRIPTION
**Pull request by gigitux**
After an internal discussion, we decided that we should insert in the testing notes all the PRs that have `User Facing Testing steps` for the final users.

This PR updates the guidelines.

### CODE REVIEW

File: .github/patch-initial-checklist.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The release pull request has been created! This checklist is a guide to follow f
 * [ ] Run `npm ci`
 * [ ] Run `npm run package-plugin:deploy`. This will create a zip of the current branch build locally.
 * [ ] Create the testing notes for the release. 
-  * [ ] For each pull request linked in the changelog grab the `User Facing Testing` notes from the PR's description. 
+  * [ ] For each pull request that belongs to the current release, grab the `User Facing Testing` notes from the PR's description. Be sure that the `Do not include in the Testing Notes is not flagged` checkbox is not flagged.
```
Comment (Aljullu) [inline .github/patch-initial-checklist.md]:
```suggestion
  * [ ] For each pull request that belongs to the current release, grab the `User Facing Testing` notes from the PR's description. Be sure that the `Do not include in the Testing Notes is not flagged` checkbox is unchecked.
```

File: .github/release-initial-checklist.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ The release pull request has been created! This checklist is a guide to follow f
 * [ ] Run `npm run package-plugin:deploy`. This will create a zip of the current branch build locally.
   *  Note: The zip file is functionally equivalent to what gets released except the version bump.
 * [ ] Create the testing notes for the release. 
-  * [ ] For each pull request linked in the changelog grab the `User Facing Testing` notes from the PR's description. 
+  * [ ] For each pull request that belongs to the current release, grab the `User Facing Testing` notes from the PR's description. Be sure that the `Do not include in the Testing Notes is not flagged` checkbox is not flagged.
```
Comment (Aljullu) [inline .github/release-initial-checklist.md]:
```suggestion
  * [ ] For each pull request that belongs to the current release, grab the `User Facing Testing` notes from the PR's description. Be sure that the `Do not include in the Testing Notes is not flagged` checkbox is unchecked.
```

**Review by Aljullu — APPROVED** [review]
I did a small suggestion to change a word, but besides that LGTM. :ship:

**Comment by gigitux** [comment]
> I did a small suggestion to change a word, but besides that LGTM. 🚢

Thanks for the great suggestion! :)

**Pull request merged**